src/test/resources/reference.conf

```diff
@@ -1,7 +1,7 @@
 include "src/main/resources/reference.conf"
 
 akka {
-  loglevel = "ERROR"
+  loglevel = "OFF"
```

Comment:
this change is not strictly necessary. The unit tests I wrote below generate expected exceptions, and akka logs them to test output in an ugly way. I turned off the logging (which quiets a few other tests but not all of them). I'm open to feedback on this.